tests/ansible/integration/transport_config/become_pass.yml

```diff
@@ -124,7 +125,7 @@
       - out.result|length == 2
       - out.result[0].method == "ssh"
       - out.result[1].method == "sudo"
-      - out.result[1].kwargs.password == "a.b.c"
+      - out.result[1].kwargs.password == "c.b.a"
```

Comment:
I don't understand. What caused this value to be reversed?

Reply:
Not sure, I tried to search through the Ansible changelog, but it appears to be how password flag handling changed between versions. This is the hosts file for `tc-become-pass-both`: `tc-become-pass-both ansible_become_password=a.b.c ansible_become_pass=c.b.a`, so it means now that `ansible_become_pass` has higher precedence than `ansible_become_password`

Reply:
I'm still concerned by this. I'll try to understand what's happened

Reply:
My best guess is the change here: https://github.com/ansible/ansible/blame/v2.10.0/lib/ansible/executor/task_executor.py#L1029-L1039 code was changed 6 and 7 months ago around handling `become_pass`, but I'm not positive. I don't really understand it either, but something changed so that `ansible_become_pass` is now the first thing accepted if both `ansible_become_pass` and `ansible_become_password` are used.

Reply:
I've adapted #762 to this branch in https://github.com/moreati/mitogen/pull/new/collectionsSupport-tox. The results of the last run were successful. They include the Mitogen Ansible tests, with strategy=linear and strategy=mitogen_linear. It appears same reversal has happened in both, so I'm content this is a correct change.

```
________________________________________________ summary ________________________________________________
  init: commands succeeded
  py27-mode_ansible-ansible2.10-distros_debian-strategy_linear: commands succeeded
  py36-mode_ansible-ansible2.10-distros_debian-strategy_linear: commands succeeded
  py37-mode_ansible-ansible2.10-distros_debian-strategy_linear: commands succeeded
  py27-mode_ansible-ansible2.10-distros_debian: commands succeeded
  py36-mode_ansible-ansible2.10-distros_debian: commands succeeded
  py37-mode_ansible-ansible2.10-distros_debian: commands succeeded
  py27-mode_mitogen-distro_centos7: commands succeeded
  py36-mode_mitogen-distro_centos7: commands succeeded
  py37-mode_mitogen-distro_centos7: commands succeeded
  report: commands succeeded
  congratulations :)
```